components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2270,6 +2270,15 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         // Note: this operation is per-user only, hence needs no resource guard
         const user = this.checkAndBlockUser("createTeam");
+
+        const mayCreateOrganization = await this.userService.mayCreateOrJoinOrganization(user);
+        if (!mayCreateOrganization) {
+            throw new ResponseError(
+                ErrorCodes.PERMISSION_DENIED,
+                "Organization-controlled users are not allowed to create new organizations",
```

Comment:
If we surface that message, it would be good to refer to "accounts" instead of "users" here. Unfortunately, in our domain model accounts are called `User`, but in comms to our users we need to distinguish between accounts and users. At least, users are not controlled through/by organizations 🤞🏻, but accounts do belong to organizations if they're created in that context (e.g. through organizational SSO)

Let's call it "Organizational accounts" or similar.

Reply:
I used "Organizational accounts" :+1: